### DISCUSSION TITLE
README内容显示不完整

### DISCUSSION THREAD

**Discussion by O1PAS** [spencerwooo/onedrive-vercel-index#777] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/115860011/195972643-fffff2d7-c9de-44b7-85d8-3e0e81379e6d.png)
如图

**Comment by spencerwooo** [comment, marked as answer]
你的网络问题。

**Reply by lihc** [reply]
@O1PAS @spencerwooo 您好：我遇到了同样的问题，将 components\previews\MarkdownPreview.tsx 文件32行中的 ${proxy ? `&proxy=true` : ''} 去掉后才显示正常，目前还不清楚路径末尾中的 “&proxy=true” 有什么作用。
对比来看：https://drive.lihaicheng.com/api/raw/?path=//README.md&proxy=true 我的链接会返回一个http状态码为500的错误，
但是项目作者的链接：https://drive.swo.moe/api/raw/?path=//README.md&proxy=true 却会正常下载。？？？？

**Reply by O1PAS** [reply]
@lihc 谢谢，这个问题困扰了我很久

**Comment by O1PAS** [comment]
试了自己的文件
![image](https://user-images.githubusercontent.com/115860011/195983740-ce59e6f2-5b4f-4913-8ee9-673ef144b0e6.png)
预览没问题
![image](https://user-images.githubusercontent.com/115860011/195983849-b1f01ca3-c247-4605-9ab7-4d24e5ec9fbd.png)
